ocs_ci/ocs/longevity.py

```diff
@@ -629,6 +666,120 @@ def stage_2(
             )
             time.sleep(delay)
 
+    def stage3(
+        self,
+        project_factory,
+        num_of_pvc,
+        num_of_obc,
+        pvc_size=None,
+        delay=60,
+        run_time=1440,
```

Comment:
any reason for this exact number ?

Reply:
It has been set to 3hrs for now and once we have a stabilized runs we will change the run time to a longer period. And this is not a strict value and can be changed based on the use case.